src/components/Video/VideoPlayerWidget.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from "react-helmet"
+
+export const VideoPlayerWidget = (props) => {
+    useEffect(() => {
+        /*global cloudinary*/
+        (function () {
+            const cld = cloudinary.Cloudinary.new({
+                cloud_name: props.cloudName, secure: props.secure, private_cdn: props.privateCdn, secure_distribution: props.cdnHost
+            });
+            return cld.videoPlayer(props.playerId, props.playerProps);
+        })();
+    }, []);
+
+    let classString = 'cld-video-player';
+    if (props.useLightSkin) {

```

Comment:
Maybe this prop could just be called `skin` in case new skins are added later. We wouldn't want to end up with multiple props like `useLightSkin`, `useDarkSkin`, `useRetroSkin` because then it's not clear what should happen if two different skins are both specified.